src/SqlFormatter.php

```diff
@@ -256,6 +256,15 @@ public function format(string $string, string $indentString = '  '): string
                 if ($token->value() === 'LIMIT' && ! $inlineParentheses) {
                     $clauseLimit = true;
                 }
+            } elseif ($token->value() === ';') {
```

Comment:
the inner code of this if branch is almost identical to the `} elseif ($token->isOfType(Token::TOKEN_TYPE_RESERVED_TOPLEVEL)) {` few lines before. 
would it make sense to group them?

Reply:
It is not worth grouping them in one if, but see #111, I have already started addressing such code duplication there. This PR should stay as is as no easy deduplication is possible and it is a bugfix.

Reply:
@goetas can you please approve this PR?

Reply:
yea, i'm sorry, I did not see how long/complex was that if branch